routes/api.php

```diff
@@ -27,3 +27,7 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+Route::middleware('auth:sanctum')->group(function () {
+    Route::post('/reset-password', [App\Http\Controllers\Auth\NewPasswordController::class, 'store']);
+    Route::post('/forgot-password', [App\Http\Controllers\Auth\PasswordResetLinkController::class, 'store']);
```

Comment:
Reset password and forgot password shouldn't require login to work. It's for the visitors mainly.